semgrep-core/matching/dune

```diff
@@ -7,6 +7,7 @@
    pfff-config
    pfff-h_program-lang
    pfff-lang_GENERIC pfff-lang_GENERIC-analyze
+   pfff-lang_js pfff-lang_js-analyze
```

Comment:
I assume this shouldn't be added to this PR?

Reply:
Actually i need this because of the ugly reference to Astjs.special

Reply:
"neat"